src/js/game/GameController.js

```diff
@@ -277,18 +277,29 @@ class GameController {
       [Phaser.Keyboard.SPACEBAR]: -2
     };
 
+    const editableElementSelected = function () {
+      return !(document.activeElement === document.body || document.activeElement === null);
```

Comment:
Hmm I'm a little worried this test might be too general. If I press "tab" on https://studio.code.org/s/minecraft/stage/1/puzzle/12, I get:

![image](https://user-images.githubusercontent.com/413693/53851231-e4441700-3f72-11e9-8087-1f3c6e635888.png)

which isn't `null` or `document.body`.